pkg/kubectl/cmd/taint_test.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	//"bytes"
+	//"net/http"
+	"reflect"
+	//"strings"
+	"testing"
+
+	"k8s.io/kubernetes/pkg/api"
+	//"k8s.io/kubernetes/pkg/api/testapi"
+	//"k8s.io/kubernetes/pkg/client/restclient"
+	//"k8s.io/kubernetes/pkg/client/unversioned/fake"
+	//"k8s.io/kubernetes/pkg/runtime"
+)
+
+func TestParseTaints(t *testing.T) {
```

Comment:
Please add some tests that demonstrate using the full command to taint one node, and to taint multiple nodes. See the 'kubectl drain' command tests in #16698 for example of how to write more a complete test for the kubectl command.